nucypher/cli/painting.py

```diff
@@ -573,3 +580,73 @@ def paint_locked_tokens_status(emitter, agent, periods) -> None:
         emitter.echo(f"{bucket_range:>9}: {box_plot:60}"
                      f"Min: {NU.from_nunits(bucket_min)} - Max: {NU.from_nunits(bucket_max)}")
 
+
+def paint_worklock_status(emitter, registry: BaseContractRegistry):
+    from maya import MayaDT
+    WORKLOCK_AGENT = ContractAgency.get_agent(WorkLockAgent, registry=registry)
+    blockchain = WORKLOCK_AGENT.blockchain
+
+    # Agency
+    token_agent = ContractAgency.get_agent(NucypherTokenAgent, registry=registry)
+
+    # Time
+    start = MayaDT(WORKLOCK_AGENT.contract.functions.startBidDate().call())
+    end = MayaDT(WORKLOCK_AGENT.contract.functions.endBidDate().call())
+    duration = end - start
+    remaining = end - maya.now()
+
+    payload = f"""
+
+Time
+======================================================
+Start Date ........ {start}
+End Date .......... {end}
+Duration .......... {duration}
+Time Remaining .... {remaining} 
+
+Economics
+======================================================            
+Max. Locked ....... {WORKLOCK_AGENT.contract.functions.minAllowableLockedTokens().call()}
+Min. Locked ....... {WORKLOCK_AGENT.contract.functions.maxAllowableLockedTokens().call()}
+Deposit Rate ...... {WORKLOCK_AGENT.contract.functions.depositRate().call()} 
+Refund Rate ....... {WORKLOCK_AGENT.contract.functions.refundRate().call()}
+Total Bids ........ {blockchain.client.get_balance(WORKLOCK_AGENT.contract_address)}
+Claimed Tokens .... {WORKLOCK_AGENT.contract.functions.allClaimedTokens().call()}
+Remaining Tokens .. {token_agent.get_balance(WORKLOCK_AGENT.contract_address)}
+
+    """
+    emitter.message(payload)
+    return
+
+
+def paint_worklock_participant_notice(emitter, bidder_address, registry):
+    worklock_agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)
+
+    obligation = f"""
+* WorkLock Participant Notice *
+-------------------------------
+
+- By participating in NuCypher's WorkLock you are committing to operating a staking NuCypher
+  node after the bidding window closes.
+
+- WorkLock token rewards are claimed in the form of a stake and will be locked for the stake duration.
+
+- WorkLock ETH deposits will be available for refund at a rate of {worklock_agent.economics.worklock_refund_rate} wei per confirmed period.
+
+- Once claiming WorkLock tokens, you are obligated to maintain a networked
+  and available Ursula-Worker node bonded to the staker address {bidder_address} for the duration 
+  of the stake(s) ({worklock_agent.economics.worklock_commitment_duration} periods).
+
+- Allow NuCypher network users to carry out uninterrupted re-encryption
+  work orders at-will without interference. Failure to keep your node online, 
+  or violation of re-encryption work orders will result in the loss of staked tokens as
+  described in the NuCypher slashing protocol.
```

Comment:
Should we provide a link to the slashing docs, perhaps http://docs.nucypher.com/en/latest/architecture/slashing.html?